src/microbe_stage/editor/MicrobeEditor.tscn

```diff
@@ -1,4 +1,4 @@
-[gd_scene load_steps=129 format=2]
+[gd_scene load_steps=108 format=2]
```

Comment:
Severe damages.

Reply:
umm, I don't think so? That should be correct